src/containers/CommunityProjects/CommunityProjectsStyles.css

```diff
@@ -0,0 +1,42 @@
+.cardFlexBox {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  margin-top: 2em;
+}
+
+.singleCardWrapper {
+  width: 30%;
+  min-width: 216px;
+  margin-right: 1.5em;
```

Comment:
we should think in `rem` rather than `em` or `px`, to accommodate people zooming for visibility, etc

Reply:
Yeah your right. Changed in 3e10178d5132a0d8c9db69e0bfc46944bbfd07d6